test/integration/verify/controls/google_compute_firewalls.rb

```diff
@@ -0,0 +1,20 @@
+title 'Firewalls Properties'
+
+gcp_project_id = attribute(:gcp_project_id, default: '', description: 'The GCP project identifier.')
+
+control 'gcp-firewalls-1.0' do
+
+  impact 1.0
+  title 'Ensure firewalls have the correct properties in bulk'
+
+  describe google_compute_firewalls(project: gcp_project_id) do
+    it { should exist }
+    its('entries.count') { should be <= 100}
+    # assume this is a development setup for a moment
+    its('firewall_names') { should include "default-allow-ssh" }
```

Comment:
You should also test firewall_ids - any property you expose.  To obtain a value to check for, you can use the "output the ID in terraform, translate to YAML, read as an attribute" approach.  AWS does this.  Other methods also work.

Reply:
added tests for the plural resource id parameters